gcloud/storage/blob.py

```diff
@@ -149,10 +149,10 @@ def public_url(self):
             bucket_name=self.bucket.name,
             quoted_name=quote(self.name, safe=''))
 
-    def generate_signed_url(self, expiration, method='GET',
-                            client=None, credentials=None,
-                            response_type=None, response_disposition=None,
-                            generation=None):
+    def generate_signed_url(self, expiration, method='GET', client=None,
+                            content_type="application/octet-stream",
+                            credentials=None, response_type=None,
+                            response_disposition=None, generation=None):
```

Comment:
Can you move `client` and `credentials` to the end? I should've noticed this before when I touched this code, sorry.